samples/weather_function/src/main/kotlin/org/example/weather_func/Weather.kt

```diff
@@ -0,0 +1,14 @@
+package org.example.weather_func
+
+/** Models the current weather for a location. */
```

Comment:
The correct doc should be styled like this:
```kotlin
/**
* Some doc
*/
```

Reply:
According to the Kotlin Coding Conventions [short comments](https://kotlinlang.org/docs/reference/coding-conventions.html#documentation-comments) can be placed on a single line.